app/src/main/java/com/example/bookwormadventuresdeluxe2/Book.java

```diff
@@ -215,6 +215,7 @@ public Status getAugmentStatus(String user)
         switch (status)
         {
             case Available:
+                return Status.Available;

```

Comment:
is this correct? Because it's crashing if i don't put the available case there 
![image](https://user-images.githubusercontent.com/39696855/98053928-f684f100-1df6-11eb-9b7f-0e5adf80a619.png)

Other issues resolved

Reply:
![image](https://user-images.githubusercontent.com/39696855/98054424-241e6a00-1df8-11eb-92b7-5873052b750b.png)

Reply:
could you elaborate on the crashing? when, what, code? I cannot replicate